src/Controls/tests/DeviceTests/Elements/Editor/EditorTests.cs

```diff
@@ -70,5 +70,22 @@ public async Task TextTransformUpdated(string text, TextTransform transform, str
 			var platformText = await GetPlatformText(handler);
 			Assert.Equal(expected, platformText);
 		}
+
+		[Fact]
+		public async Task MaxLengthIsReadOnlyValueTest()
+		{
+			Editor editor = new Editor();
+			editor.MaxLength = 0;
+
+			await InvokeOnMainThreadAsync(() =>
+			{
+				var handler = CreateHandler<EditorHandler>(editor);
+				var platformControl = GetPlatformControl(handler);
+				editor.MaxLength = 10;
```

Comment:
Maybe you also need to check that it is read only before changing the max length.

Reply:
can do!